이상한-문자-만들기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    let newText = s.split(" ");
+    newText = newText.map((item) => item.split(""));
+
+    newText = newText.map((item1, index1) =>
+        item1.map((item2, index2) =>
+            index2 % 2 ? item2.toLowerCase() : item2.toUpperCase()
+        )
+    );
+
+    const answer = newText.map((item, index) => item.join("")).join(" ");
+    return answer;
+}
```

Comment:
```suggestion
function solution(s) {
    const newText = s.split(" ").map((item) => item.split(""));

    const weirdText = newText.map((item1, index1) => {
        return item1.map((item2, index2) => {
            return index2 % 2 ? item2.toLowerCase() : item2.toUpperCase()
        })
    });

    return weirdText.map((item, index) => item.join("")).join(" ");
}
```

2, 3번 라인은 한 라인으로 묶어서 처리해도 될거같고 참조타입은 변수선언을 한번 더 하는 일이 생기더라고 const로 선언해주는게 좋아요 그리고 화살표함수에서 줄바꿈이 생기면 중괄호로 묶어주는게 더 가독성이 좋다고 생각합니다.

Reply:
return 쓰기 싫어서 {} 안 썼는데 다음부터 사용하겠습니다ㅠㅠ